zellij-server/src/os_input_output.rs

```diff
@@ -156,6 +159,9 @@ pub trait ServerOsApi: Send + Sync {
     fn send_to_client(&self, msg: ServerToClientMsg);
     /// Adds a sender to client
     fn add_client_sender(&self);
+    fn send_to_temp_client(&self, msg: ServerToClientMsg);
```

Comment:
Sure. Each client has a separate router thread. A temp client is the one that hasn't been added to the client list yet, so only the router thread has access to send to it. This may be the case when the client is trying to attach but is unable to do so. Or in future, this may be used if the client is there to only query some information (Eg: Active sessions' information) and exit.

I'll add a comment to the code as well.